blogposts/more-intuitive-code-search-on-sourcegraph-dot-com.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Search the top 10,000 open source repositories on Sourcegraph.com"
+description: "How Sourcegraph.com became easier to use, by setting the default search scope to the top ten thousand open source repositories on GitHub."
+author: Issac Trotts, Software Engineer at Sourcegraph
+publishDate: 2019-08-16T00:00-11:55
+tags: [
+ blog
+]
+slug: Search-the-top-10000-open-source-repositories-Sourcegraph-com
+heroImage: /blog/sourcegraph-com-usage.png
+published: true
+---
+
+<p style="text-align: center">
+ <img src="/blog/sourcegraph-com-usage.png" />
+ <br/><small>Sourcegraph.com stats as of August 2019</small>
+</p>
+
+Wouldn't it be awesome if you could scope a code search to find usage examples from the top ten thousand open source repositories on GitHub? You can, using [Sourcegraph.com](https://sourcegraph.com), and we've now made it easier to use!
+
+## Using Sourcegraph.com like a regular search engine
+
+[Sourcegraph.com](https://sourcegraph.com) has always allowed developers to browse any open source repository from GitHub, but it wasn't possible to perform a search without adding a `repo` filter to limit the number of repositories that needed to be searched (private Sourcegraph instances don't have this limitation). This meant [Sourcegraph.com](https://sourcegraph.com) was missing a crucial use case—finding usage examples for a library, class, method, or function, from a set or reputable, and predefined open source repositories.
+
+Let's say you've recently heard about Google's [Distroless container image](https://github.com/GoogleContainerTools/distroless), and are curious about its usage. Previously, without specifying any repositories, your search would've returned this:
+
+![](/blog/too-many-matching-repos.png)
+
+After some [Sourcegraph.com](https://sourcegraph.com) specific code changes ([#4959](https://github.com/sourcegraph/sourcegraph/pull/4959/files) and [#5062](https://github.com/sourcegraph/sourcegraph/pull/5062/files)
+), Sourcegraph (by default), will now search the top 10,000 open source repositories from GitHub (ordered by watch count) if a [`repo` filter](https://docs.sourcegraph.com/user/search/queries) was not provided. This means a query for a [distroless based Dockerfile](https://sourcegraph.com/search?q=file:Dockerfile+FROM+gcr.io/distroless) now returns a useful set of results to explore.
```

Comment:
There is a newline here before the `)` that shouldn't be.

Reply:
Fixed!